src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -1291,6 +1291,14 @@ def handle(self, *args, **options):
         self._remove_deprecated_credential_kinds()
         enable_redis_prefix()
 
+    def content_type_model(self):
+        try:
+            # DAB RBAC migrated to a custom type model, try to use that here
+            return apps.get_model("dab_rbac", "DABContentType")
+        except LookupError:
```

Comment:
Hi @AlanCoding If we finally depend on a new revision of DAB that includes this change, this fallback should not be necessary. Am I right?

Reply:
Correct.